packages/logs/src/boot/logs.entry.ts

```diff
@@ -36,9 +36,11 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
   const globalContextManager = createContextManager()
   const customLoggers: { [name: string]: Logger | undefined } = {}
 
-  const beforeInitSendLog = new BoundedBuffer<[LogsMessage, Context]>()
+  const beforeInitSendLog = new BoundedBuffer()
   let sendLogStrategy = (message: LogsMessage, currentContext: Context) => {
-    beforeInitSendLog.add([message, currentContext])
+    beforeInitSendLog.add<[LogsMessage, Context]>([message, currentContext], ([message, context]) =>
+      sendLogStrategy(message, context)
+    )
   }
```

Comment:
Storing arguments in `BoundedBuffer` seems to be avoidable. What about simply passing a closure?

```ts
  let sendLogStrategy = (message: LogsMessage, currentContext: Context) => {
    beforeInitSendLog.add(() => sendLogStrategy(message, context))
  }
```

Reply:
thanks, way simpler indeed!